packages/components/src/Chip/Chip.types.d.ts

```diff
@@ -0,0 +1,35 @@
+import React, { PropsWithChildren } from "react";
+
+export interface ChipProps extends PropsWithChildren {
+  /** Accessible label, which can be different from the primary label. */
+  readonly ariaLabel?: string;
```

Comment:
Sorry for the last minute comment! Not a deal breaker but would you mind using multiline comments for the types? I personally find it helps with readability and follows our props pattern in other components 😃 

```suggestion
  /**
   * Accessible label, which can be different from the primary label.
   */
  readonly ariaLabel?: string;

```